Add Standard for Public Code introduction course (#23)
publiccodenet/projects: index.md

This is a living document, and [we welcome your feedback](http://standard.publiccode.net/CONTRIBUTING.html).

#### [Introduction course](https://citybycity.academy/course/standards-for-smart-cities)

> Together with Open & Agile Smart Cities we have created a free and open online introduction course to the Standard for Public Code.

Reviewer: ```suggestion
Together with [Open & Agile Smart Cities](https://oascities.org/) we have created a free and open online introduction course to the Standard for Public Code.
```

> It's a self study course and it offers a certification of completion for those who finishes all the modules.

Reviewer: ```suggestion
It's a self study course and offers a certification of completion to those who finish all the modules.
```
